gulpfile.js

```diff
@@ -30,9 +30,6 @@
       .pipe(tsProject())
       .pipe(sourcemaps.write('.'))
       .pipe(gulp.dest('distrib/lib'));
-  }, function () {
-    return gulp.src('./external/**/*')
-      .pipe(gulp.dest('./distrib/lib/external/'));
```

Comment:
So we hand not other external dependencies, other than the OCSP libs?

Reply:
Correct.